README.md

```diff
@@ -1 +1,33 @@
-# engagement-monitor
\ No newline at end of file
+# Engagement Monitor
+
+![Engagement Monitor Logo](https://github.com/akshatvg/engagement-monitor/blob/master/Backend/engagement/static/img/Logo.png "Engagement Monitor Logo")
+
+### What is Engagement Monitor?
+Engagement Monitor is a web application which helps analyse active or disinterested members of a WhatsApp group.
+
+
+### How to use Engagement Monitor?
+1) Go to group or individual chat settings on WhatsApp in your mobile phone and choose the 'Export Chat' option and save the .txt file anywhere on your device.
+2) Open Engagement Monitor and upload the file after registering and logging in.
+3) View each member's total count of messages and past counts in numbers as well as in the form of graphs and charts.
+
+
+### Advantages of Engagement Monitor?
+1) Find out the most active members in a WhatsApp group.
+2) Find out who spams the most in a WhatsApp group.
+3) Keep track of inactive people who rarely/ never reply to or acknowledge messages.
+4) Easy to use UI & UX.
+5) Saves time checking statistics of member activity.
+
```

Comment:
@akshatvg 
Can you please add some **screenshots** or screen **recordings** too?

Reply:
@vinitshahdeo will add them all once the project is complete and ready.